src/operations.test.js.md

```diff
@@ -3845,22 +3892,9 @@ Generated by [AVA](https://ava.li).
                   branch: {
                     type: 'string',
                   },
-                  created_at: {
```

Comment:
Wondering if this would turn this release into a breaking changes? 🤔 

It is my understanding that we’re not validating body parameters against the open-api spec (besides checking its format) - https://github.com/netlify/js-client/blob/master/src/methods/body.js#L2, so probably not?

CC @ehmicky @erezrokah

Reply:
If someone is doing `api.createSiteBuildHook()` and expecting those properties to be returned, this would be a breaking change for them.
Should we make a major release?

Reply:
That's true, however it's my understanding that the fields would not be there before... So one could argue that we didn't really end up breaking anything? 🤷‍♂️

Anyway, tbh, if we have so many questions whether this is a breaking change or not I would rather do a major release. Semver numbers are cheap 😄

Reply:
We had two major releases last month, we should consider how to slow that cadence. Doesn’t necessarily need to hold up this release, but we should zoom out and consider what else to include in the release (if anything).

Reply:
If we feel like this should not break any of our users, we could also make a minor release?

Reply:
I don't belive this will break any code, as the behaviour here hasn't actually changed 👍 I'm going to merge this as a feature and let the `release-please` bot do its thing.